bot/controllers/poll.js

```diff
@@ -0,0 +1,91 @@
+const BaseController = require('../baseController.js');
+const Command = require('../baseCommand.js');
+const util = require('apex-util');
+
+class PollController extends BaseController {
+  constructor(message) {
+    // Call BaseController constructor
+    super(message);
+
+    // Aliasing 'this' as controller to allow for binding in actions
+    const controller = this;
+
+    // Array of all commands, see baseCommand.js for prototype
+    this.commands = [
+      new Command(
+        '!poll',
+        '!poll <ask-a-question> <option_1> <option_2> <option_3>',
+        'Posts a Question users can vote on',
+        'Posts a Question users can vote on, use "-" to separate words in your question',
+        this.pollAction.bind(controller),
+      ),
+    ];
+  }
+
+  pollAction() {
+    const { message } = this;
+    const question = message.parsed[1];
+    const questionSpaced = question.split('-').join(' ');
+    const content = message.content;
```

Comment:
I'm picking up a linting issue here, even though it seems to have gone through the tests successfully. However, I do recommend de-structuring it:
```
const { content } = messsage;
```